### ISSUE TITLE
`dyn_class` method panics when trying to remove a class that isn't associated with the HTML element.

### ISSUE THREAD

**Issue by whereistejas** [sycamore-rs/sycamore#332]
**Describe the bug**
Removing a class that's not associated with a html tag causes `panic`.

**To Reproduce**
```rust
let some_signal = Signal::new(true);
a().dyn_class("some class", some_signal.handle()).build()
```

**Expected behavior**
The following code should not panic if the CSS class is not associated with that element.

**Screenshots**

**Environment**

- Sycamore: `sycamore = { version = "0.7.1", features = ["experimental-builder-html"] }`
- Browser: Chrome (Vivaldi)
- OS: MacOS Montenery

**Additional context**
Error log:
```
Uncaught (in promise) Error: `unwrap_throw` failed
    at imports.wbg.__wbindgen_throw ((index):4744)
    at wasm_bindgen::throw_str::h01afea94b45683c2 (cae1I5KPOdh.wasm:0x533584)
    at <sycamore::generic_node::dom_node::DomNode as sycamore::generic_node::GenericNode>::remove_class::hd7b224c5fbc0f0da (cae1I5KPOdh.wasm:0x4e4381)
    at sycamore::builder::agnostic::NodeBuilder<G>::dyn_class::{{closure}}::h55760c369a542475 (cae1I5KPOdh.wasm:0x5099ca)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hb1893d1a8d4d4f0e (cae1I5KPOdh.wasm:0x531663)
    at std::thread::local::LocalKey<T>::with::hf1bb6d6c899aa17e (cae1I5KPOdh.wasm:0x2f0d72)
    at sycamore_reactive::create_child_scope_in::h53bf0a342129f6b4 (cae1I5KPOdh.wasm:0x4c479c)
    at std::thread::local::LocalKey<T>::with::hdcabeed18826778e (cae1I5KPOdh.wasm:0x278339)
    at sycamore_reactive::effect::_create_effect::{{closure}}::h2feea1fc6915e656 (cae1I5KPOdh.wasm:0x531c49)
    at sycamore_reactive::effect::_create_effect::h420ffa21234f3978 (cae1I5KPOdh.wasm:0x357aa0)
    at sycamore::builder::agnostic::NodeBuilder<G>::dyn_class::h43e02838d6d9ea50 (cae1I5KPOdh.wasm:0x4b6c6d)
```

**Comment by tqwewe** [comment]
I also get this error with this code:
```rust
#[component(Breadcrumbs<G>)]
pub fn breadcrumbs(items: Vec<BreadcrumbItem>) -> View<G> {
    let items_len = items.len();
    let item_views = View::<G>::new_fragment(
        items
            .into_iter()
            .enumerate()
            .map(|(index, item)| {
                let classes = format!(
                    "breadcrumb-item {}",
                    if index == items_len - 1 {
                        "breadcrumb-item-selected"
                    } else {
                        ""
                    }
                );

                view! {
                    li(class="breadcrumb-item") {
                        a(href=(item.href)) { (item.label) }
                    }
                }
            })
            .collect(),
    );

    view! {
        nav(aria-label="Breadcrumb") {
            ol {
                (item_views)
            }
        }
    }
}
```

**Comment by lukechu10** [comment]
According to [the spec](https://dom.spec.whatwg.org/#ref-for-dom-domtokenlist-remove%E2%91%A0), it's because their is a whitespace in the class name. We should probably do the same thing we did for `add_class` which is to split the classes at whitespace and add all of them.

**Comment by lukechu10** [comment]
I believe this is now fixed by c3a6ba408ab2961e899c6bd2b0d87cb713a1bda1
